### DISCUSSION TITLE
Tag system with customizable options for Special and Hyper Attacks

### DISCUSSION THREAD

**Discussion by Paul45mec** [ikemen-engine/Ikemen-GO#985] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Natively we only have the characters changing, that's already great, but wouldn't there be the possibility of adding more command options for customization in different styles like Kof, Marvel vs capcom 2, Kizuna encounter and Street fighter X tekken and etc...?

A pre-configuration when you can customize the style according to your game.

I thought of something simple and accessible to everyone, in the past we used unoshe unotag which was a very useful and customizable tool.

### Describe the solution you'd like

TAG SYSTEM CONFIGURATION ( IN UNOSHE UNOTAG YOU HAVE 3 DIFFERENT STYLES TO CHOOSE)

Q/W = Change the characters normally

B+Y = Super move on ground / Super move antiaerial if the opponent is in the air.
full circle + C+Z = Hyper move combination between 2,3 or 4 characters.

C+Z = Change when the character jump on the screen with a kick or punch like MVC Series (optional)

Taunt & Voice during the swap or after executing a super move like the Marvel vs capcom series. 

### Additional context

The tag code in Ikemen GO 0.98.2 uses these states:
StateTagEnteringScreen = 5600
StateTagLeavingScreen = 5610
StateTagWaitingOutside = 5611
StateTagJumpingIn = 5620
StateTagLanding = 5621

### Search terms

Tag system with customizable options for Special and Hyper Attacks

**Comment by K4thos** [comment, marked as answer]
Not gonna happen.. Default tag is meant to be used as a simple base for custom tag implementation. Just like you don't use default screenpack, you should start looking for other options, if base tag is not enough.
